ERCS/erc-XXXX.md

```diff
@@ -13,60 +13,64 @@ requires: 712, 1193
 
 ## Abstract
 
-A new `secureRequest` method in addition to EIP-1193 `request` for dApp to interact with extension wallets in a tamperproof way, preventing any MITM from modifying the payload sent to the wallet.
-This will allow for dApps to sign their interaction with EIP-1193, and for wallet to visibly display secure interaction, therefor improving the end user experience by explicitly displaying safety
-visual cues such as a padlock.
+Introduces a new RPC method `wallet_signedRequest` that enables dapps to interact with wallets in a tamperproof manner via "signed requests".
```

Comment:
I'm proposing the name `wallet_signedRequest`, but not completely sold on it.

Reply:
i'm ok with any name. Was going with `secure` since it is what is used in TLS (Transport layer security), but signed is more explicit about signing/integrity vs encryption/confidentiality

Reply:
should we mention explicitly 1193 so the reader is immediately aware of what part of the stack would change ?

Reply:
I actually decided to remove EIP-1193 and 712 as requirements and added a "References" section at the bottom of the doc. I did update the line in question to mention that the method should be implemented by wallets. It can be agnostic of the specific provider interface.